tests/topotests/pytest.ini

```diff
@@ -1,16 +1,29 @@
 # Skip pytests example directory
 [pytest]
 norecursedirs = .git example-test example-topojson-test lib docker
+
+# Markers
+#
+# Please consult the documentation and discuss with TSC members before applying
+# any changes to this list.
 markers =
-	babel: Tests that run against BABEL
-	bfd: Tests that run against BFDD
-	eigrp: Tests that run against EIGRPD
-	isis: Tests that run against ISISD
-	ldp: Tests that run against LDPD
-	ospf: Tests that run against OSPF( v2 and v3 )
-	pbr: Tests that run against PBRD
-	pim: Tests that run against pim
-	rip: Tests that run against RIP, both v4 and v6
+	babeld: Tests that run against BABELD
```

Comment:
why are we adding the `d` at the end?  I see no need for this.  I'm testing a protocol or a feature not a daemon name. imo.  That is why I choose the names I did for the first round

Reply:
Because this is for the CI system, and we will test based on changes in daemon subdirectories. If you want to test for a protocol, just use ``pytest -v -k pim`` to select all objects, modules and files containing the substring "pim" in their names.